app/views/foods/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Results: <%= @foods.totalhits %> </h1>
+
+<%= @foods.food_info.each do |food| %>
+<%= @description %>
+<%= @gtinupc %>
```

Comment:
These instance variables would only be passed to the view if they were set in the controller. You don't have access to instance variables you set in your `FoodIndex` without an attr_reader, and then you would use the method dot syntax.